lib/impure/db_mysql.nim

```diff
@@ -115,38 +132,81 @@ when false:
       binding: seq[MYSQL_BIND]
     discard mysql_stmt_close(stmt)
 
-proc dbQuote*(s: string): string =
-  ## DB quotes the string.
-  result = "'"
-  for c in items(s):
-    if c == '\'': add(result, "''")
-    else: add(result, c)
-  add(result, '\'')
+proc dbQuote*(s: SQLInput): string =
+  ## Database sanitizes the SQLInput
+  case s.kind:
+  of SQLStringKind:
+    result = "'"
```

Comment:
I wonder if it might be worth using `ewStringOfCap`](https://nim-lang.org/docs/system.html#newStringOfCap,Natural) with the length of `s.str` + 2 initially?